compose/install/index.md

```diff
@@ -0,0 +1,39 @@
+---
+description: How to install Docker Compose
+keywords: compose, orchestration, install, installation, docker, documentation
+title: Install Docker Compose
+toc_max: 2
+---
+
+This page contains information on how to install Docker Compose. You can run Compose on macOS, Windows, and 64-bit Linux.
+
+
+## Prerequisites
+
+* Docker Compose requires Docker Engine.
+* Docker Compose plugin requires Docker CLI.
+
+## Install Compose
```

Comment:
If you've gone with my suggestions above I'd remove this header

Reply:
I need the anchor, or at least I think I do, lol. So would "Compose installation scenarios (summary)" work for you?

Reply:
Yes, that would work

Reply:
PS: Changed it Compose Installation Scenario to address this and your next comment in this conversation.